gravitee-apim-repository/gravitee-apim-repository-mongodb/src/main/java/io/gravitee/repository/mongodb/management/MongoTagRepository.java

```diff
@@ -66,6 +66,16 @@ public Optional<Tag> findByIdAndReference(String tagId, String referenceId, TagR
         return Optional.ofNullable(mapper.map(tag));
     }
 
+    @Override
+    public Set<Tag> findByIdsAndReference(Set<String> tagIds, String referenceId, TagReferenceType referenceType) throws TechnicalException {
+        LOGGER.debug("Find tags by IDs and reference [{}, {}, {}]", tagIds, referenceId, referenceType);
+
+        final List<TagMongo> tags = internalTagRepo.findByIdInAndReferenceIdAndReferenceType(tagIds, referenceId, referenceType);
+
+        LOGGER.debug("Find tag by IDs and reference[{}, {}, {}] - Done", tagIds, referenceId, referenceType);
```

Comment:
It seems a duplicate log. We already have one L71. Am I missing something?

Reply:
There's an "s" after ID since it's a list :P

Reply:
The relevant was `- Done` that I didn't see 🙈